src/simulation/onboardDataHandling/storageUnit/simpleStorageUnit.cpp

```diff
@@ -79,3 +79,12 @@ void SimpleStorageUnit::integrateDataStatus(double currentTime){
     this->previousTime = currentTime;
     return;
 }
+
+/*! Adds a specific amount of data to the storedData vector once
+ @param data //Data to be added to the "STORED DATA" partition
+ @return void
+ */
+void SimpleStorageUnit::setDataBuffer(long long int data){
+    std::string partitionName = "STORED DATA";
+    SimpleStorageUnit::DataStorageUnitBase::setDataBuffer(partitionName, data);
+}
```

Comment:
This file is missing a new line at its end. You can see that github is alerting you of this fact with the little red circle containing a dash, just to the left above this comment.